src/controller/constraints.php

```diff
@@ -184,15 +184,11 @@ protected function checkComment(string $contents, &$errors)
      */
     protected function redirectNoAdmin()
     {
-        try {
-            // If I do not follow admin, return to the article page 
-            if ($_SESSION['rankConnectedUser'] !== self::USER_RIGHT_ADMIN) {
-                new Exception($_SESSION['error'] = self::MESSAGE_NO_ADMIN);
-                header('location:Articles-Page1');
-                exit;
-            }
-        } catch (Exception $e) {
-            $e->getMessage();
+        // If I do not follow admin, return to the article page
+        if ($_SESSION['rankConnectedUser'] !== self::USER_RIGHT_ADMIN) {
```

Comment:
idem peut être voir pour faire une fonction qui check seulement si ton user est admin et dans ton controller tu rediriges via ton $this->render

Reply:
OK